pupil_src/shared_modules/audio_playback.py

```diff
@@ -164,7 +164,7 @@ def _setup_output_audio(self):
             self.audio_sync = self.pa_stream.get_output_latency()
             self.audio_reported_latency = self.pa_stream.get_output_latency()
 
-        except ValueError:
+        except (ValueError, OSError):
             self.pa_stream = None
```

Comment:
Actually, let's handle OSError specifically
```suggestion
        except ValueError:
            self.pa_stream = None
        except OSError:
            self.pa_stream = None
            import traceback

            logger.warning("Audio found, but playback failed (#2103)")
            logger.debug(traceback.formt_exc())
```